go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -393,6 +393,7 @@ func (be *BuiltinBackupEngine) backupFiles(ctx context.Context, cnf *Mycnf, mysq
 		BackupManifest: BackupManifest{
 			BackupMethod: builtinBackupEngineName,
 			Position:     replicationPosition,
+			BackupTime:   backupTime.Format(time.RFC3339),
```

Comment:
Shouldn't we convert to UTC?

Reply:
this is a good point. we were relying on the caller to be passing in UTC time, but I notice that vtbackup isn't doing that:
```		backupTime := time.Now()
```

Reply:
I have fixed this to convert to UTC.